velox/functions/lib/string/tests/CMakeLists.txt

```diff
@@ -22,4 +22,5 @@ target_link_libraries(
   gtest
   gtest_main
   gflags::gflags
-  glog::glog)
+  glog::glog
+  fast_float)
```

Comment:
Is this needed?

Reply:
Yes, The reason is similar to the above.